code/modules/client/preference_setup/loadout/loadout_eyes.dm

```diff
@@ -147,3 +147,17 @@
 	goggles["goggles, tactical (Zavodskoi)"] = list(/obj/item/clothing/glasses/safety/goggles/goon/zavod, "Zavodskoi Interstellar")
 	goggles["goggles, tactical (Idris)"] = list(/obj/item/clothing/glasses/safety/goggles/goon/idris, "Idris Incorporated")
 	gear_tweaks += new /datum/gear_tweak/path/faction(goggles)
+
+/datum/gear/eyes/medical_goggles
+	display_name = "goggles selection (medical)"
```

Comment:
please make it clear that these are functional and not just regular goggles.
suggestion: "HUD goggles selection (medical)" or "medical HUD goggles selection".